components/dashboard/src/components/UsageBasedBillingConfig.tsx

```diff
@@ -32,8 +32,10 @@ interface Props {
     hideSubheading?: boolean;
 }
 
+// Guard against multiple calls to subscripe (per page load)
+let didAlreadyCallSubscripe = false;
```

Comment:
Small typo i think, unless it was a clever combo of subscription and stripe 😄 :

```suggestion
let didAlreadyCallSubscribe = false;
```

Reply:
That was definitely just a typo - fixed!